frontend/views/pages/DesignSystem.vue

```diff
@@ -1149,6 +1174,15 @@ export default {
         el.link.classList.remove('last')
       })
       this.articles[last].link.classList.add('last')
+    },
+    async saveIn2500ms () {
+      const dumbPromiseThatTakes2500ms = () => new Promise((resolve, reject) => {
+        setTimeout(() => {
+          resolve()
+        }, 2500)
+      })
+
+      await dumbPromiseThatTakes2500ms()
```

Comment:
Async/await is unnecessary here, can be written simply:

```js
saveIn2500ms () {
  return new Promise((resolve, reject) => {
    setTimeout(resolve, 2500)
  })
}
```

Reply:
I explicitly used async/await to be more similar to the real-world...

Reply:
This is real world too, and way simpler. You're creating a promise in either case. Might as well not over-complicate things. Save lines please!

Reply:
okay, changed.